src/components/feedback/components/forms.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ControlTextarea from '@mapbox/mr-ui/control-textarea';
+import classnames from 'classnames';
+
+// character limit for the feedback textarea
+export const feedbackLimit = 1000;
+export const feedbackMinimum = 3;
```

Comment:
@HeyStenson @colleenmcginnis What do you think about this minimum for required textarea? I carried this number over from https://github.com/mapbox/dr-ui/blob/920eb460aa7911eb49f692253b5104e3c5b25f60/src/components/feedback/feedback.js#L243. I think increasing it could add friction, but could help enforce more words from the user.

Reply:
The shortest meaningful word I can think of is "No" but if someone added that as text input there's no scenario in which that would actually be useful information ... so I think 3 characters is good! If we find that we're getting poor quality feedback, we could up the value ⬆️

Reply:
I like that plan! Leaving as is! Thanks!